### PR TITLE
AVASUP-442: Change default label Customs Duty and Import Tax

### PR DESCRIPTION
**Pull request by v-okrushko**

### CODE REVIEW

File: Model/Sales/Pdf/Tax.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        CollectionFactory $ordersFactory,
+        Config $taxConfig,
+        array $data = []
+    ) {
+        $this->_taxConfig = $taxConfig;
+        parent::__construct($taxHelper, $taxCalculation, $ordersFactory, $data);
+    }
+
+    /**
+     * Check if tax amount should be included to grandtotal block
```
Comment (marynychsv) [inline Model/Sales/Pdf/Tax.php:43]: redundant code

File: etc/di.xml [hunk 2]
```diff
@@ -303,4 +303,5 @@
             <argument name="taxCalculation" xsi:type="object">ClassyLlama\AvaTax\Framework\Interaction\TaxCalculation\Proxy</argument>
         </arguments>
     </type>
+    <preference for="Magento\Tax\Model\Sales\Pdf\Tax" type="ClassyLlama\AvaTax\Model\Sales\Pdf\Tax"/>
```
Comment (marynychsv) [inline etc/di.xml:306]: is there another way to do this, like with plugin or so?
thank you

File: Model/Sales/Pdf/Tax.php (hunk 1 above)
Comment (v-okrushko) [reply Model/Sales/Pdf/Tax.php]: file was removed

File: etc/di.xml (hunk 2 above)
Comment (v-okrushko) [reply etc/di.xml]: replaced with plugin

**Pull request merged**